app/views/devise/registrations/new.html.slim

```diff
@@ -66,6 +66,10 @@ h2 Sign up
         = b.label
 
   .form-group
+        = f.collection_select :highest_education, FormalEducation.all, :id, :name, { prompt: true }, class: "form-control"
+  br
```

Comment:
remove br. This is not needed. If we really need a spacing between highest education and Preferred Centers, then we do it in the css using `margin-bottom`